sdk/formrecognizer/azure-ai-formrecognizer/tests/test_send_request.py

```diff
@@ -42,3 +43,88 @@ def test_get_resource_details(self, client, **kwargs):
         assert received_info2["customDocumentModels"]["count"] == received_info1["customDocumentModels"]["count"]
         assert received_info2["customDocumentModels"]["limit"] == received_info1["customDocumentModels"]["limit"]
+        
+        # test with full url
+        request = HttpRequest(
+            method="GET",
+            url=f"{client._endpoint}/formrecognizer/info?api-version=2022-08-31",
+            headers={"Accept": "application/json"},
+        )
+        result = client.send_request(request)
+        received_info3 = result.json()
+        assert received_info3["customDocumentModels"]["count"] == received_info1["customDocumentModels"]["count"]
+        assert received_info3["customDocumentModels"]["limit"] == received_info1["customDocumentModels"]["limit"]
+        
+        # test with v2 api version
+        request = HttpRequest(
+            method="GET",
+            url="v2.1/custom/models?op=summary",
+            headers={"Accept": "application/json"},
+        )
+        result = client.send_request(request)
+        received_info4 = result.json()
+        assert received_info4
+        assert received_info4["summary"]["count"] == 0
+        assert received_info4["summary"]["limit"]
+        
+        # test with v2 api version with full url
+        request = HttpRequest(
+            method="GET",
+            url=f"{client._endpoint}/formrecognizer/v2.1/custom/models?op=summary",
+            headers={"Accept": "application/json"},
+        )
+        result = client.send_request(request)
+        received_info5 = result.json()
+        assert received_info5["summary"]["count"] == received_info4["summary"]["count"]
+        assert received_info5["summary"]["limit"] == received_info4["summary"]["limit"]
+
+    @FormRecognizerPreparer()
+    @FormTrainingClientPreparer(client_kwargs={"api_version": "2.1"})
+    @recorded_by_proxy
+    def test_get_account_properties_v2(self, client):
+        set_bodiless_matcher()
+        request = HttpRequest(
+            method="GET",
+            url="custom/models?op=summary",
+            headers={"Accept": "application/json"},
+        )
+        result = client.send_request(request)
+        received_info1 = result.json()
+        assert received_info1
+        assert received_info1["summary"]
+        assert received_info1["summary"]["count"] == 0
+        assert received_info1["summary"]["limit"] == 250
+        
+        # test with full url
+        request = HttpRequest(
+            method="GET",
+            url=f"{client._endpoint}/formrecognizer/v2.0/custom/models?op=summary",
+            headers={"Accept": "application/json"},
+        )
+        result = client.send_request(request)
+        received_info3 = result.json()
+        assert received_info3["summary"]["count"] == received_info1["summary"]["count"]
+        assert received_info3["summary"]["limit"] == received_info1["summary"]["limit"]
+        
+        # test with new api version
```

Comment:
This example really shows that this request is still targeting `v2.1` (and the operation doesn't work on that version), right?
```suggestion
        # relative URLs can't override the API version on 2.x clients
```